prusti-common/src/vir/ast/expr.rs

```diff
@@ -160,6 +164,18 @@ impl fmt::Display for Expr {
                     .collect::<Vec<String>>()
                     .join(", "),
             ),
+            Expr::DomainFuncApp(ref function, ref args, ref _pos) => write!(
+                f,
+                "{}({})",
+                function,
```

Comment:
`function.name`? I think this prints the full function signature; I'm not sure if this is what you wanted.

Reply:
Right. This needs to change anyway as DomainFuncApp should mirror the signature of FuncApp.